coordinator.ts

```diff
@@ -38,6 +38,14 @@ app.post("/:appId/login/anonymous", (req, res) => {
   const token = `e30.${Buffer.from(JSON.stringify(user)).toString("base64")}`;
   res.json({ token });
 });
+app.post("/:appId/login/nickname", (req, res) => {
+  const { appId } = req.params;
+  const { nickname } = req.body;
+  const id = crypto.createHash('md5').update(`${appId}:${nickname}`).digest('hex').slice(0,12);
```

Comment:
Does the user input need to be sanitized here before adding it to the hash?

For example only allowing some characters to avoid script injection on the client side?

Perhaps this doesn't matter for a local coordinator, but I hope it's taken care of in the production coordinator.

Reply:
The thought process when originally writing this was to allow a "nickname" to be easily ported across devices/browsers/sessions. So I deterministically generated the id to enable that use case.

But @hpx7 already pointed out that, for apps that want that behavior, they can easily implement that by using `user.name` as the primary key instead of `user.id`. I'm going to update this to match the random id generation logic of `anonymous` auth.

Thanks for the feedback!